web_console/web-console.adoc

```diff
@@ -7,22 +7,14 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {product-title} web console is a user interface accessible from a web browser.
-Developers can use the web console to visualize, browse, and manage the contents
-of projects.
+The {product-title} web console is a user interface accessible from a web browser. You can use the web console to visualize, browse, and manage the contents of projects.
 
 == Prerequisites
 
-* JavaScript must be enabled to use the web console. For the best experience, use
-a web browser that supports
-link:http://caniuse.com/#feat=websockets[WebSockets].
-* Review the link:https://access.redhat.com/articles/4128421[OpenShift Container
-Platform 4.x Tested Integrations] page before you create the supporting
-infrastructure for your cluster.
+* JavaScript must be enabled to use the web console. For the best experience, use a web browser that supports link:http://caniuse.com/#feat=websockets[WebSockets].
```

Comment:
in OpenShift 4.19, console team changed their build target to es2021: https://github.com/openshift/console/blob/main/frontend/packages/console-dynamic-plugin-sdk/CHANGELOG-webpack.md#4190-prerelease1---2025-02-17

Thus, a web browser running web console should support es2021 for the best experience. Should this be written in the documentation?